spec/models/flower_shop_spec.rb

```diff
@@ -4,8 +4,6 @@
   describe 'relationships' do
     it { should have_many :flowers }
   end
-
-  describe 'instance methods' do
```

Comment:
Did you mean to remove this `describe` block? github makes it hard to see whether you have the right # of `end` statements now, so I'm assuming this was on purpose, but it looks odd to me here.